frontend/app/module/data_entry/page/DataEntryHomePage.test.tsx

```diff
@@ -103,21 +103,19 @@ describe("DataEntryHomePage", () => {
     } satisfies ElectionStatusResponse);
 
     // render and expect the initial status to be fetched
-    const { rerender } = renderDataEntryHomePage();
+    renderDataEntryHomePage();
     expect(await screen.findByText("Welk stembureau ga je invoeren?")).toBeVisible();
     expect(screen.queryByText("Alle stembureaus zijn ingevoerd")).not.toBeInTheDocument();
 
     // unmount DataEntryHomePage, but keep the providers as-is
-    rerender(
+    render(
       <ElectionProvider electionId={1}>
         <ElectionStatusProvider electionId={1}>
           <></>
         </ElectionStatusProvider>
       </ElectionProvider>,
     );
-    await waitFor(() => {
-      expect(screen.queryByText("Welk stembureau ga je invoeren?")).not.toBeInTheDocument();
-    });
+    expect(await screen.findByText("Welk stembureau ga je invoeren?")).toBeVisible();
```

Comment:
The purpose of the test is to test if a rerender using the same provider component triggers a re-fetch (i.e. navigate away from the data entry home page and then back again). The change on L111 to an explicit `render` breaks the test, because this renders an additional set of new components (it does not clear the result of the previous render!) and so it will always refetch. This does not align with the purpose of this test.

Reply:
Sorry about that - I misunderstood what was happening. I will restore the original test.